orchagent/aclorch.cpp

```diff
@@ -301,6 +301,12 @@ const set<sai_acl_action_type_t>& AclTableType::getActions() const
     return m_aclAcitons;
 }
 
+bool AclTableType::addAction(sai_acl_action_type_t action)
+{
+    m_aclAcitons.insert(action);
+    returne true;
```

Comment:
typo ?

Reply:
@arlakshm Thanks for catching this. Fixed the typo.